src/mixins/createTabbedMixin.ts

```diff
@@ -195,9 +196,9 @@ const createTabbedMixin: TabbedMixinFactory = createCachedRenderMixin
 
 			function getTabChildVNode(tab: TabbedChild): (VNode | string)[] {
 				const tabListeners = getTabListeners(tabbed, tab);
-				const nodes = [ h('div.tab-label', { onclick: tabListeners.onclickTabListener }, [ tab.state.label ]) ];
+				const nodes = [ h('div.' + css['tab-label'], { onclick: tabListeners.onclickTabListener }, [ tab.state.label ]) ];
```

Comment:
String literal would be preferred over concatenation (I know we aren't 100% consistent on that yet)...

```typescript
const nodes = h(`div.${css['tab-label']}`)
```